packages/berry-pnpify/package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "@berry/pnpify",
-  "version": "0.1.1",
-  "main": "./sources/index.ts",
-  "bin": "./sources/boot-dev.js",
+  "version": "0.1.2",
```

Comment:
For version bumps the ideal is to use `yarn version <strategy> --deferred`, as that allows us to aggregate multiple changes into a single version bump (otherwise I need to remember that pnpify shouldn't be bumped again until I publish it).

Reply:
Done